x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -40,6 +40,18 @@ export const DETECTION_RULE_ALERTS_STATUS_API_CURRENT_VERSION = '1';
 export const DETECTION_RULE_RULES_API_CURRENT_VERSION = '2023-10-31';
 
 export const CLOUD_SECURITY_POSTURE_PACKAGE_NAME = 'cloud_security_posture';
+
+export const CDR_MISCONFIGURATIONS_DATA_VIEW_NAME = 'Latest Cloud Security Misconfigurations';
+export const CDR_MISCONFIGURATIONS_DATA_VIEW_ID_PREFIX =
+  'security_solution_latest_misconfigurations';
+export const CDR_MISCONFIGURATIONS_INDEX_PATTERN =
+  'logs-*_latest_misconfigurations_cdr,logs-cloud_security_posture.findings_latest-default';
```

Comment:
I discussed this with @kfirpeled, and he prefers being explicit to avoid confusing developers who might read the code and assume there are multiple indices. If we need to change the logic in the future, we can always edit the current index pattern.